frontend/packages/yki/src/components/registration/examSession/PublicExamSessionListingFilters.tsx

```diff
@@ -18,28 +50,155 @@ export const PublicExamSessionFilters = () => {
     });
   };
 
-  // TODO Fixme
-  const searchButtonDisabled = false;
-  const handleEmptyBtnClick = scrollToSearch;
-  // eslint-disable-next-line no-console
-  const handleSearchBtnClick = () => console.log('empty btn clicked..');
+  const { filters, municipalities } = useAppSelector(examSessionsSelector);
+  const filteredExamSessions = useAppSelector(selectFilteredPublicExamSessions);
```

Comment:
Mun mielestä on aika selkeä että tää komponentti hakee itse redux-tilasta tarvitsemansa jutut.

Reply:
Tällä hetkellä komponentti saa propsina vain lokaaliin (ei redux) liittyviä callbackeja.

Reply:
Niin, eipä tähän mitään suoria sääntöjä varmaan ole. Reduxin osalta ehkä oon ajatellut itse että kun kerran kyseessä on tavallaan globaali tila johon mikä tahansa komponentti pääsee selektorien kautta käsiksi, niin sama sitten hakea tieto mahdollisimman lähellä paikkaa missä sitä tarvitaan. Tähän ehkä voi myös Reduxin guidelinesien nähdä kannustavan, joskin ehkä mitään täysin suoraa ei asiasta oo sanottukaan.

https://redux.js.org/style-guide/#call-useselector-multiple-times-in-function-components
https://redux.js.org/style-guide/#connect-more-components-to-read-data-from-the-store

Reply:
Tää muuten ihan mielenkiintoinen huomio tuolta:
```
For example, rather than just connecting a <UserList> component and reading the entire array of users, have <UserList> retrieve a list of all user IDs, render list items as <UserListItem userId={userId}>, and have <UserListItem> be connected and extract its own user entry from the store.
```
Näin ei tietystikään olla nyt itse tehty vaikkei nyt niin vakavaa